.pylintrc

```diff
@@ -1,44 +1,95 @@
 # Generated using `.pylintrc. --generate-rcfile` and then modified.
 
-[MASTER]
+[MAIN]
+
+# Analyse import fallback blocks. This can be used to support both Python 2 and
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
+
+# Load and enable all available extensions. Use --list-extensions to see a list
+# all available extensions.
+#enable-all-extensions=
+
+# In error mode, messages with a category besides ERROR or FATAL are
+# suppressed, and no reports are done by default. Error mode is compatible with
+# disabling specific errors.
+#errors-only=
+
+# Always return a 0 (non-error) status code, even if lint errors are found.
+# This is primarily useful in continuous integration scripts.
+#exit-zero=
 
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
+extension-pkg-allow-list=
+
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code. (This is an alternative name to extension-pkg-allow-list
+# for backward compatibility.)
 extension-pkg-whitelist=
 
-# Add files or directories to the blocklist. They should be base names, not
-# paths.
-#
-# Files under alembic are generated and are nonconforming.
-ignore=alembic
```

Comment:
Why get rid of stuff like this?

Reply:
This `pylintrc` was generated automatically with the latest version of `pylint`, I am working on manually merging our old settings back so that we can have:
1. The default settings of the latest `pylint`
2. The same setting from us in the past.